core/capabilities/ccip/launcher/launcher.go

```diff
@@ -351,21 +354,25 @@ func createDON(
 		lggr.Infow("Not a member of this DON and not a bootstrap node either, skipping", "donID", don.ID, "p2pID", p2pID.String())
 		return nil, nil
 	}
+	var aggErr error
 	p := make(pluginRegistry)
+
 	for _, config := range configs {
 		digest, err := ocrtypes.BytesToConfigDigest(config.ConfigDigest[:])
 		if err != nil {
-			return nil, fmt.Errorf("digest does not match type %w", err)
+			aggErr = errors.Join(aggErr, fmt.Errorf("digest does not match type %w", err))
+			continue
 		}
 
 		oracle, err := oracleCreator.Create(ctx, don.ID, cctypes.OCR3ConfigWithMeta(config))
 		if err != nil {
-			return nil, fmt.Errorf("failed to create CCIP oracle: %w for digest %x", err, digest)
+			aggErr = errors.Join(aggErr, fmt.Errorf("failed to create CCIP oracle: %w for digest %x", err, digest))
```

Comment:
```suggestion
			aggErr = errors.Join(aggErr, fmt.Errorf("failed to create CCIP oracle: %w for digest %x", err, digest[:]))
```

Since `digest` is a `[32]byte` I've seen this show up in logs as a super large integer rather than the expected hex value - I think adding the slice notation might fix it? Worth checking in a test.

Reply:
`digest` is of `ConfigDigest` type which has `String()` function defined https://github.com/smartcontractkit/libocr/blob/7324c200a6b0fff57449fc90025eb8b99df5e735/offchainreporting2plus/types/config_digest.go#L98

Therefore, it's always printed nicely. Checked in the test manually

Reply:
Maybe switch the format verb to `%s` then? Cuz this is what I see in our production logs as the error:

```json
{
  "level": "error",
  "ts": "2025-10-29T15:59:06.131Z",
  "caller": "launcher/launcher.go:147",
  "msg": "Failed to tick",
  "version": "2.29.0@00848b1",
  "err": "failed to process diff: processAdded: call createDON 23: failed to create CCIP oracle: failed to create CCIPProviders: failed to get chain selector from relay ID 26888 and family evm: invalid chain id 26888 for evm for digest 30303061656263393431663534626235373935643936323531613035313335333163613238323366663565343830323535613736313838313363363631633763",
  "stacktrace": "github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/launcher.(*launcher).monitor\n\t/chainlink/core/capabilities/ccip/launcher/launcher.go:147"
}
```

Notice the part after the `for digest`